vm-lifecycle.ps1

```diff
@@ -93,23 +93,27 @@ function Wait-ScaleTask {
 
 Write-Host "Create VM"
 $json = @{
-    name = "VMLifeycleTest"
-    description = "An example created for testing the rest API"
-    mem = 4GB
-    numVCPU = 4;
-    blockDevs = @(
-        @{
-            capacity = 1GB
-            type = 'VIRTIO_DISK'
-            cacheMode = 'WRITETHROUGH'
-        }
-    )
-    netDevs = @(
-        @{
-            type = 'VIRTIO'
-        }
-    )
-} | ConvertTo-Json
+	dom = @{
+	    name = "VMLifeycleTest"
+	    description = "An example created for testing the rest API"
+	    mem = 4GB
+	    numVCPU = 4;
+	    blockDevs = @(
+		@{
+		    capacity = 1GB
+		    type = 'VIRTIO_DISK'
+		    cacheMode = 'WRITETHROUGH'
+		}
+	    )
+	    netDevs = @(
+		@{
+		    type = 'VIRTIO'
+		}
+	    )
+	}
+	options = @{
```

Comment:
I think a comment could be added on the role this serves. With that missing, this is just something arbitrary left blank. It could be as simple as a comment mentioning a value that could be set here. That would communicate that it is ok to omit the value.